pkg/proto/chat/chat.proto

```diff
@@ -304,6 +314,7 @@ message SearchUserInfoResp {
 service chat {
   //编辑个人资料 自己或者管理员调用
```

Comment:
Please remove the Chinese comments. Thank you!

Reply:
ok

Reply:
ok